### PR TITLE
GUI/sky background2

### PR DESCRIPTION
**Pull request by AlemSnyder**
This renders stars with triangles.

It is much faster than sky background (one).

### CODE REVIEW

File: src/gui/data_structures/screen_data.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    glGenBuffers(1, &vertex_buffer_);
+    glBindBuffer(GL_ARRAY_BUFFER, vertex_buffer_);
+    glBufferData(
+        GL_ARRAY_BUFFER, vertices.size() * sizeof(glm::vec3),
+        vertices.data(), GL_STATIC_DRAW
+    );
+
+    num_vertices_ = 4;
+
+    std::vector<unsigned short> indices;
```
Comment (egelja) [inline src/gui/data_structures/screen_data.cpp:24]: FIXED WIDTH

**Review by egelja — CHANGES_REQUESTED** [review]
preliminary comments until we format

File: make_stars.py
[the remark this answers is not in the document]
Comment (AlemSnyder) [reply make_stars.py:1]: I'm not going to get a python formatter for three files

File: src/entity/mesh.hpp [hunk 2]
```diff
@@ -151,9 +151,8 @@ generate_mesh(T voxel_object) {
 
     std::vector<glm::i8vec3> indexed_normals;
     // mesh off set
-    glm::i32vec3 center = voxel_object.get_offset();
-    glm::u32vec3 dims = voxel_object.get_size();
-    glm::ivec3 offset(center.x, center.y, center.z);
+    VoxelOffset center = voxel_object.get_offset();
+    VoxelSize object_geometry = voxel_object.get_size();
```
[the remark this answers is not in the document]
Comment (AlemSnyder) [reply src/entity/mesh.hpp:155]: its not worth it

[the remark this answers is not in the document]
Comment (egelja) [reply src/entity/mesh.hpp]: Yes it is

**Comment by AlemSnyder** [comment]
Something something look at imgui

**Pull request closed without merging**